pom.xml

```diff
@@ -469,6 +493,11 @@
         <configuration>
           <instructions>
             <_exportcontents>
+              org.apache.commons.*;
```

Comment:
I'm sure that I added this because of failures. Double-checked this in the preview mode and deployed with both OData V4 and V2.
Changed to:
```
 <_exportcontents>
    com.fasterxml.aalto.stax.*;
    io.cdap.plugin.sap.*
</_exportcontents>
```

I assume other exports were needed because of some dependencies during development, that are no longer needed.

`com.fasterxml.aalto.stax.*;` is added to solve:
```
Caused by: java.util.ServiceConfigurationError: javax.xml.stream.XMLInputFactory: Provider com.fasterxml.aalto.stax.InputFactoryImpl not found
	at java.util.ServiceLoader.fail(ServiceLoader.java:239) ~[na:1.8.0_144]
	at java.util.ServiceLoader.access$300(ServiceLoader.java:185) ~[na:1.8.0_144]
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:372) ~[na:1.8.0_144]
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404) ~[na:1.8.0_144]
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480) ~[na:1.8.0_144]
	at javax.xml.stream.FactoryFinder$1.run(FactoryFinder.java:353) ~[na:1.8.0_144]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_144]
	at javax.xml.stream.FactoryFinder.findServiceProvider(FactoryFinder.java:341) ~[na:1.8.0_144]
	... 42 common frames omitted
```